### ISSUE TITLE
Link previews v 1.5.0

### ISSUE THREAD

**Issue by MillesC** [pedroslopez/whatsapp-web.js#156]
The Link previews is not running.
I am using v 1.5.0

**Comment by pedroslopez** [comment]
What do you mean? You're not getting the preview?

Can you make sure you've updated the version and ran `npm install`, and try sending, for example, https://github.com ?

I just tested again and it seems to work.

**Comment by joaomirandasa** [comment]
@pedroslopez no, it's necessary improve your method, i fix solution by myself, if you're trying to send with actual code only sender can see object build on his browser but user(receiver) on phone never see that, it's necessary to add an paramether and remove one IF

**Check preview.subtype = "url"**

My solution:
```node
if (options.linkPreview) {
            delete options.linkPreview;
            const link = window.Store.Validators.findLink(content);
            if (link) {
                const preview = await window.Store.Wap.queryLinkPreview(link.url);
                preview.preview = true;
                preview.subtype = "url";
                options = { ...options, ...preview};
            }
}```

**Comment by pedroslopez** [comment]
@joaomirandasa 

Hmm... this seems to work for me
![image](https://user-images.githubusercontent.com/4368928/79084288-e5607f80-7d00-11ea-9d17-32aa049c76fd.png)

On both sides I get the preview (bot's message is on the left). I can add those props to the object, but I wonder why it works for me without them.

**Comment by joaomirandasa** [comment]
I can't precise why that's happening but for me and my instances, only this change make work, probably related to deviceType or something like that.

**Comment by pedroslopez** [comment]
@joaomirandasa are you on android? Also, why did you remove the status check? It's to prevent including the property if the request fails.

@MillesC can you try and check if it works for you by specifying subtype="url"?

**Comment by joaomirandasa** [comment]
I've tested on android and iOS at the same time, for me only this changes make appear on both sides..

I saw that check as unnecessary, WhatsApp clean the object by himself

**Comment by pedroslopez** [comment]
Addressed in bab54ae. New release (1.5.1) coming shortly. Thanks for reporting!
